lib/sqewer/perform_override.rb

```diff
@@ -0,0 +1,17 @@
+# This module enables the acceptance of kwargs for the perform method inside
+# of ActiveJob Execute. 
+module PerformWithKeywords
+  def perform_now
+    begin
```

Comment:
I would've left the `def` - `rescue` - `end` on the same indent level. It looks prettier 😊

Reply:
In this case we can use a naked rescue even.